simulation.py

```diff
@@ -0,0 +1,73 @@
+"""
+A function to simulate alpha
+"""
+
+import copy
+import importlib
+import sys
+from thousandaire.constants import TRADING_CONFIG, DATA_LIST_ALL
+from thousandaire.data_loader import DataLoader
+from thousandaire.simulator import Simulator
+
+DATASET = 'price_dataset'
+
+def initialize():
+    """
+    Initialize data we get.
+    """
+    raw_data = DataLoader(DATA_LIST_ALL).get_all()
+    workdays = raw_data['workdays']
+    invest_data = {index : value
+                   for index, value in raw_data.items() if index != 'workdays'}
+    return_data = {}
+    for key, value in workdays.items():
+        return_data[key] = copy.deepcopy(invest_data)
+        for dataset in return_data[key].values():
+            dataset.set_workdays(value)
+    #Notice, this is a little different from other dataset in return_data.
+    return_data['workdays'] = workdays
+    return return_data
+
+def set_data(raw_data, data_list, target, region):
+    """
+    Use to filter data we need.
+    In the future may put in simulator to pickle needed data while
+    using multi-process.
+    """
+    workdays = raw_data['workdays'][region]
+    if target in raw_data[region].keys():
+        target_data = copy.deepcopy(raw_data[region][target])
+    else:
+        raise KeyError("Key not found")
+    simulate_data = {name : copy.deepcopy(raw_data[region][name])
+                     for name in data_list if name in raw_data[region].keys()}
+    data = {
+        'workdays' : workdays,
+        'target_data' : target_data,
+        'simulate_data' : simulate_data}
+    return data
+
+def simulation(alpha_path, raw_data):
+    """
+    Act the process of simulating.
+    """
+    setting_module = importlib.import_module(alpha_path).AlphaSettings()
+    data = set_data(
+        raw_data,
+        setting_module.data_list,
+        TRADING_CONFIG[(setting_module.target[0],
+                        setting_module.target[1])][DATASET],
+        setting_module.target[1])
+    simulator = Simulator(setting_module, data)
+    results = simulator.run()
+    return results
+
+if __name__ == '__main__':
+    ARGS = sys.argv
```

Comment:
This is not a constant. Please don't use upper variable names.

Reply:
But in pylint, it said it should be upper case.
https://docs.google.com/document/d/1sI2QRgBw7jhX8MQhkn-ZtG4Ccw2GBwDKB-yJIGAMSVU/edit?usp=sharing

Reply:
That's because any variables defined here are global variables, so for other functions they are something constant. Putting the code to a `main` function can fix this issue naturally.

Reply:
Oh! While put them into _def main( )_ it will be legal.